input/pagecontent/reference.md

```diff
@@ -259,7 +257,7 @@ This syntax is also used with CodeableConcepts (see [Assignments with the Codeab
 FSH provides a shorthand that allows three aspects of a Quantity to be set simultaneously:
 
 * The numerical quantity (`Quantity.value`)
-* The units of measure (`Quantity.code`)
+* The units of measure (`Quantity.system` and `Quantity.code`)
```

Comment:
You really need the code _and system_ to properly specify the units.  Even when you use the `'{ucum unit}'` syntax, we're stuffing UCUM into `Quantity.system` for you.